packages/contracts-bedrock/contracts/test/L1CrossDomainMessenger.t.sol

```diff
@@ -295,173 +291,6 @@ contract L1CrossDomainMessenger_Test is Messenger_Initializer {
         assertEq(L1Messenger.failedMessages(hash), true);
     }
 
-    // relayMessage: Should revert if the recipient is trying to reenter with the
```

Comment:
Its really annoying that we have duplicate tests for both L1 and L2 - we should really be testing the base contract directly, there is nothing to do as part of this PR but just wanted to note this